indico/web/client/js/react/components/EmailListField.jsx

```diff
@@ -47,6 +47,7 @@ const EmailListField = props => {
       placeholder={Translate.string('Please enter an email address')}
       additionLabel={Translate.string('Add email') + ' '} // eslint-disable-line prefer-template
       onChange={handleChange}
+      icon=""
```

Comment:
That is this doing? Seems a bit strange to set icon to an empty string..

Reply:
It's to hide the dropdown icon on the `EmailListField`. I don't think it makes sense to show it since it's supposed to act more like a text input...

Reply:
OK makes sense